app/app/api/bot/route.ts

```diff
@@ -0,0 +1,38 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+export async function POST(req: NextRequest) {
+  const body = await req.json()
+
+  const message = body?.message
+  const chatId = message?.chat?.id
+  const text = message?.text
+
+  if (text === '/start' && chatId) {
+    const telegramApiUrl = `https://api.telegram.org/bot${process.env.TELEGRAM_BOT_TOKEN}/sendMessage`
+
+    await fetch(telegramApiUrl, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        chat_id: chatId,
+        text: 'Welcome! Tap the button below to open the app.',
```

Comment:
Instead of showing the button, could it just open the app immediately?

Reply:
@prodalex Which option do you prefer?

Reply:
this is how it looks like when you open the storacha_bot for the first time
<img width="515" alt="image" src="https://github.com/user-attachments/assets/57c3439c-8908-43e8-bbff-bc0f35f54565" />
This start button should open the TG mini app.

In consecutive usage just typing /start to get it started is enough. we do not need to over-engineer this right now.

<img width="511" alt="image" src="https://github.com/user-attachments/assets/0bfb8377-5eee-4487-9adf-9b7b2539d5c6" />

Reply:
awesome! so when you pressed the button, the "/start" text was sent, yes?